src/Calculator/GraphingSettings.xaml

```diff
@@ -0,0 +1,136 @@
+<Page
+    x:Class="CalculatorApp.Controls.GraphingSettings"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:CalculatorApp"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    DataContextChanged="GraphingCalculator_DataContextChanged"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Loaded="OnLoaded"
+    >
+    <Page.Resources>
+        <SolidColorBrush x:Key="ListMediumControlBackgroundBrush" Color="{StaticResource SystemListMediumColor}" />
+        <SolidColorBrush x:Key="ControlPointerOverBorderBrush" Color="LightGray" />
+        <SolidColorBrush x:Key="ControlSelectedBackgroundBrush" Color="LightGray" />
+        <SolidColorBrush x:Key="BaseMediumLowPressedBackgroundBrush" Color="{StaticResource SystemBaseMediumLowColor}" />
+        <SolidColorBrush x:Key="ChromeWhiteHighControlSelectedForegroundBrush" Color="#FFFFFFFF" />
+        <SolidColorBrush x:Key="BackgroundColorBrush" Color="CornflowerBlue" />
+        <SolidColorBrush x:Key="ForegroundColorBrush" Color="White" />
+
+        <Style x:Key="FilterRadioButtonStyle" TargetType="RadioButton">
+            <Setter Property="MinHeight" Value="44" />
+            <Setter Property="MinWidth" Value="0" />
+            <Setter Property="Padding" Value="4,4" />
+            <Setter Property="HorizontalAlignment" Value="Stretch" />
+            <Setter Property="FocusVisualMargin" Value="0" />
+            <Setter Property="Template">
+                <Setter.Value>
+                    <ControlTemplate TargetType="RadioButton">
+                        <Grid
+                        Name="LayoutRoot"
+                        Margin="0,0"
+                        Background="{ThemeResource RadioButtonBackground}">
+                            <Border
+                            Name="AccessibilityBorder"
```

Comment:
Spacing is off, do you have XamlStyler installed? It should handle most of this stuff automatically, check out instructions on [README.md](https://github.com/microsoft/calculator/blob/master/README.md)

Reply:
I didn't, was just using default VS handling.  Fixed.